magicctapipe/io/io.py

```diff
@@ -528,6 +670,92 @@ def load_train_data_files(
 
     return data_train
 
+def load_train_data_files_tel(input_dir, config, offaxis_min=None, offaxis_max=None, true_event_class=None):
```

Comment:
the new function varies very little from the old one, why not simply specify as an argument if you should split by telescopes or by combinations?
also the config in this and get_stereo_events functions is used only to get the telescopes combinations, so why not giving the combinations as an argument (this would also simplify a lot integration with the old code)

Reply:
As above, this is due to the glueing of old and new pipeline. Now we still use combo splitting, but the new RF-DL2 scripts will only use telescope splitting. But again we will discuss about this

Reply:
ok, same here, I will close it for now, and we rediscuss with the other PR. I think we would be able to also preserve the possibility of the old behaviour with only small additions to the new code